Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
That… doesn’t make _any_ sense to me. How can the list of packages tested contain _only_ a `/vendor/` package which has been explicitly filtered out?

Can you look into how `$PACKAGES` is computed, and perhaps compare that with `make --just-print test`?